### PR TITLE
Added quartz verb

### PR DESCRIPTION
**Pull request by ImperatorS79**
quartz is needed for Elite:Dangerous launcher to start, see [https://bugs.winehq.org/show_bug.cgi?id=43162](https://bugs.winehq.org/show_bug.cgi?id=43162)

### CODE REVIEW

File: Engines/Wine/Verbs/quartz/script.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    remove(this.prefixDirectory + "/TMP/");
+    
+    this.regsvr32().install("quartz.dll");
+    
+    this.overrideDLL()
+        .set("native, builtin", ["quartz"])
+        .do()
+    
+    return this;
+
```
Comment (madoar) [inline Engines/Wine/Verbs/quartz/script.js]: please remove this empty line

File: Engines/Wine/Verbs/quartz/script.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+include(["Engines", "Wine", "Engine", "Object"]);
+include(["Utils", "Functions", "Net", "Resource"]);
+include(["Utils", "Functions", "Filesystem", "Files"]);
+
+Wine.prototype.quartz = function(){
```
Comment (madoar) [inline Engines/Wine/Verbs/quartz/script.js]: Maybe add some documentation about this function/verb

Comment (ImperatorS79) [reply Engines/Wine/Verbs/quartz/script.js]: I don't see well where you want some documentation. It's quite explicit and the other verbs doesn't have any "documentation"

Comment (madoar) [reply Engines/Wine/Verbs/quartz/script.js]: You're basically right.
What I've meant is that you add a javadoc like comment to the head of the method.
This comment should contain the content of the verb and its source (ideally with an url).
In this case, the comment could be `Inspired from winetricks quartz` combined with the url to the source of the winetricks quartz implementation.

Comment (ImperatorS79) [reply Engines/Wine/Verbs/quartz/script.js]: Is what I've just commit what you want ?

Comment (madoar) [reply Engines/Wine/Verbs/quartz/script.js]: Almost, I think the link to the winetricks implementation of the script would be better, which I think is: https://github.com/Winetricks/winetricks/blob/63bc6dbe612d017a0cb6bf6e4cde265162d75bca/src/winetricks#L8523

Comment (ImperatorS79) [reply Engines/Wine/Verbs/quartz/script.js]: Yes, sorry, that wasn't what I wanted to copy. I have used https://raw.githubusercontent.com/Winetricks/winetricks/master/src/winetricks

Comment (ImperatorS79) [reply Engines/Wine/Verbs/quartz/script.js]: It's corrected now

**Comment by plata** [comment]
@madoar I think @ImperatorS79 cannot merge to master so you should do it when you accept a PR.

**Comment by madoar** [comment]
I didn't merge this yet, because I want to let you take a look at the PRs, because you're more knowledgeable with the scripts.
Sorry I should have written that.

**Comment by plata** [comment]
If it works and the script looks ok, I think it's perfectly fine if you merge it.

**Pull request merged**